whitepaper/main.tex

```diff
@@ -27,6 +27,11 @@
 	}
 	\end{spacing}
 
+	{
+		\pagestyle{empty}
+		\include{typography}
```

Comment:
i'm unsure if typography or preface should come first ?

Reply:
imo typography, but not sure either